frontend/src/pages/ContactUs/ContactUs.jsx

```diff
@@ -91,34 +90,27 @@ export const ContactUs = (props) => {
     setFormErrors(errors);
   };
   const submitContactFormData = async (formData) => {
-    var api = `${END_POINT}/contactus`;
     setIsLoading(true);
-    return await fetch(api, {
-      method: "POST",
-      body: JSON.stringify(formData),
-      headers: {
-        "Content-type": "application/json",
-      },
-    })
-      .then((res) => res.json())
-      .then((data) => {
-        if (
-          data.message === "Database Error" ||
-          data.message === "Sendgrid Error"
-        ) {
-          setIsLoading(false);
-          setContactToast(data.message);
-          setContactToastStatus("error");
-        } else {
-          setIsLoading(false);
-          setContactToastStatus("success");
-          setContactToast(data.message);
-          setOpenSubmitContactSuccess(true);
-        }
-      })
-      .catch((err) => {
-        console.info(err);
-      });
+    try {
+      const data = await postContactUs(formData);
+      if (
+        data.message === "Database Error" ||
+        data.message === "Sendgrid Error"
+      ) {
+        setIsLoading(false);
+        setContactToast(data.message);
```

Comment:
toast message and status can also be moved to the service layer na? those who want to use can use them and rest can ignore, it will also maintain consistency in toast message.

Reply:
Added